prime-router/docs/onboarding-users/transport/rest.md

```diff
@@ -0,0 +1,160 @@
+# REST Transport
+## Introduction
+ReportStream supports four popular transport protocols:  Simple Object Access Protocol (SOAP), Resource State Transfer (REST), Secure File Transfer Protocol (SFTP), and Azure Blob. The goal of this document is to explain how to configure REST Transport for STLTs (a/k/a receivers). As well as provide guidance on how: 1) ReportStream REST Transport works, 2) to prepare a key for authentication, 3) to configure receiver's REST transport, and 4) test/check that a connection is successful. Please see ABC document on how to configure other transports.
+## Assumptions ([refer to SFTP transport documentation for this section](https://github.com/CDCgov/prime-reportstream/blob/master/prime-router/docs/onboarding-users/transport/sftp.md))
```

Comment:
Maybe it's just me, but I feel like it's weird to refer to a document for a different transport type for assumptions.

Reply:
agreed. maybe just copy relevant information straight from the SFTP document to here?

Reply:
Did copied the whole assumptions from SFTP.mp to here.